man/systemd.service.xml

```diff
@@ -77,6 +77,19 @@
     with SysV</ulink> document.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Service Templates</title>
+
+    <para>It is possible for Systemd services to take a single argument via the
+    <literal><replaceable>service</replaceable>@<replaceable>argument</replaceable></literal>
```

Comment:
hmm, this is confusing I think, since you drop the `.service` suffix here... I think this would be a lot clearer if the `.service` suffix is mentioned here explicitly, after all we are in the `.service` man page here...

Maybe:

`<literal><replaceable>service</replaceable>@<replaceable>argument</replaceable>.service</literal>`